code/modules/mob/living/carbon/xenomorph/castes/runner/abilities_runner.dm

```diff
@@ -113,6 +114,16 @@
 	update_button_icon()
 
 /datum/action/ability/xeno_action/evasion/action_activate()
+	//Since both the button and the evasion extension call this proc directly, check if the cooldown timer exists
+	//The evasion extension removes the cooldown before calling this proc again, so use that to differentiate if it was the player trying to cancel
+	if(evade_active && cooldown_timer)
+		if(TIMER_COOLDOWN_CHECK(src, COOLDOWN_EVASION_ACTIVATION))
+			owner.balloon_alert(owner, "Accidental deactivation guard!")	//Little message to let the player know and not think it is a bug that their evasion didn't turn off
```

Comment:
this is extremely wordy for a balloon alert.

Reply:
its fine, just probably has no reason to exist. don't see why the user needs to be alerted about this in the first place

Reply:
```suggestion
```
?

Reply:
I guarantee someone will say the evasion cancelling is buggy or laggy because they tried to do it too fast. Informing the user something is not working the way they intended matters.

Reply:
Then someone will just tell them they're being a goober, this is clutter

Reply:
Ok.